lib/origen/generator/flow.rb

```diff
@@ -7,7 +7,7 @@ def create(options = {}, &block)
         # Origen.deprecated 'Origen core will no longer support program generation in v1, add the origen_testers plugin to your application to fix this warning'
         # Refresh the target to start all settings from scratch each time
         # This is an easy way to reset all registered values
-        Origen.app.reload_target!
+        Origen.app.reload_target! if options[:reload_target]
```

Comment:
Not sure about this, seems like a breaking change.

Would prefer to see it written the other way around, so that by passing in a new option it will inhibit the reloading.

Reply:
Also not sure if this actually works, does it?

This method is partially through transitioning to OrigenTesters, I think it should override what you have here - https://github.com/Origen-SDK/origen_testers/blob/master/lib/origen_testers/origen_ext/generator/flow.rb

Reply:
@ginty i backed out that change